crates/ruff_linter/src/rules/refurb/rules/single_item_membership_test.rs

```diff
@@ -25,6 +25,11 @@ use crate::fix::edits::pad;
 /// 1 == 1
 /// ```
 ///
+/// ## Fix safety
+///
+/// When the right-hand side is a string, the fix is marked as unsafe.
+/// This is because `c in "a"` is true both when `c` is `"a"` and when `c` is the empty string.
```

Comment:
```suggestion
/// This is because `c in "a"` is true both when `c` is `"a"` and when `c` is the empty string,
/// so the fix can change the behavior of your program in these cases.
```